foodsaving/stores/models.py

```diff
@@ -34,7 +34,7 @@ class Feedback(BaseModel):
     given_by = models.ForeignKey('users.User', on_delete=models.CASCADE, related_name='feedback')
     about = models.ForeignKey('PickupDate')
     weight = models.PositiveIntegerField(blank=True, null=True)
-    comment = models.CharField(max_length=settings.DESCRIPTION_MAX_LENGTH)
+    comment = models.CharField(max_length=settings.DESCRIPTION_MAX_LENGTH, blank=True)
```

Comment:
@id-gue I couldn't submit a feedback without comment through the API. It seems necessary to set `blank=True`, per https://docs.djangoproject.com/en/1.11/ref/models/fields/#blank

I undid the changes to `weight`, it seems necessary to have both `blank` and `null` there: to allow for empty input and to store empty input as null/none.